app/main.py

```diff
@@ -64,27 +88,29 @@ def make_middleware() -> List[Middleware]:
         Middleware(
             EventHandlerASGIMiddleware,
             handlers=[local_handler]
+        ),
+        Middleware(
+            LogMiddleware, 
+            logger=uvicorn_logger,
+            file_logger=file_logger
         )
     ]
 
+app = FastAPI(
+    openapi_url=f"{ settings.API_V1_STR }/openapi.json",
+    middleware=make_middleware()
+)
 
-logger = logging.getLogger(__name__)
-config_path=Path(__file__).with_name("logging_config.json")
-
-def create_app() -> FastAPI:
-    app = FastAPI(
-        openapi_url=f"{ settings.API_V1_STR }/openapi.json",
-        middleware=make_middleware()
+@app.exception_handler(CustomException)
+async def custom_exception_handler(request: Request, exc: CustomException):
+    content = { "error_code": exc.error_code, "message": exc.message }
+    if settings.DEV_PHASE == DevPhase.DEV:
+        content["stack"] = exc.stack
+    return JSONResponse(
+        status_code=exc.code,
+        content=content,
```

Comment:
why move this?

Reply:
Because the way it was done before meant the `custom_exception_handler`  function showed up as unused by the linter. Plus, it was unnecessary to have the method inside a wrapper method when we can just do it this way.

Reply:
I moved it back to the way it was